internal/pkg/term/progress/ecs.go

```diff
@@ -149,6 +158,27 @@ func (c *rollingUpdateComponent) renderFailureMsgs(out io.Writer) (numLines int,
 	return renderComponents(out, components)
 }
 
+func (c *rollingUpdateComponent) renderAlarms(out io.Writer) (numLines int, err error) {
+	if len(c.alarms) == 0 {
+		return 0, nil
+	}
+	header := []string{"Name", "State"}
+	var rows [][]string
+	for _, a := range c.alarms {
+		rows = append(rows, []string{
+			a.Name,
+			prettifyAlarmState(a.Status),
+		})
+	}
+	table := newTableComponent(color.Faint.Sprintf("Alarms"), header, rows)
+	table.Padding = c.padding
+	nl, err := table.Render(out)
+	if err != nil {
+		return 0, fmt.Errorf("render alarms table: %w", err)
+	}
+	return nl, err
```

Comment:
nit: can we do like the above method where we have create an empty line before the `Alarms` table?
```go
components := []Renderer {
  &singleLineCompoenent{}, // Add an empty line before renderering alarms.
  table,
}
return renderComponents(out, components)
```